doc/sphinx/proof-engine/ltac2.rst

```diff
@@ -355,6 +356,15 @@ Ltac2 Definitions
       the `f` in the body is resolved dynamically. This is witnessed by
       the second re-definition.
 
+.. tacn:: Set @qualid := @ltac2_expr
+
+   This expression redefines a previous ``mutable`` definition at runtime.
+   Contrarily to its command equivalent above, it allows to replace a definition
+   from inside a tactic, and will take effect when the expression is evaluated.
+   This construct has type :n:`unit`, and the class of definitions allowed to be
+   replaced at runtime is currently restricted to monomorphic ones. It does not
+   feature the :n:`as` clause of the command, since the same effect can be
+   obtained using a simple let-binding.
```

Comment:
```suggestion
   Redefines a previous ``mutable`` definition at runtime.
   Unlike its command equivalent above, it allows replacing a definition
   from inside a tactic.  The redefinition takes effect when the expression is evaluated.
   This construct has type :n:`unit`.  Currently only monomorphic definitions can be
   replaced at runtime. It does not
   feature the :n:`as` clause of the command, since the same effect can be
   obtained using a simple let-binding.
```